tasks/rh-sign-image/tests/test-rh-sign-image-single-component-plr-alreadysigned.yaml

```diff
@@ -0,0 +1,144 @@
+                  }
+                },
+                "sign": {
+                  "request": "simple-signing-pipeline",
+                  "requestType": "internal-pipelinerun",
+                  "configMapName": "signing-config-map"
+                }
+              }
+              EOF
+              cat > "$(workspaces.data.path)/release_plan_admission.json" << EOF
+              {
+                "spec": {
+                  "pipeline": {
+                    "pipelineRef": {
+                      "resolver": "git",
+                      "params": [
+                        {
+                          "name": "url",
+                          "value": "https://localhost.git"
+                        },
+                        {
+                          "name": "revision",
+                          "value": "main"
+                        },
+                        {
+                          "name": "pathInRepo",
+                          "value": "pipelines/abc/abc.yaml"
+                        }
+                      ]
+                    },
+                    "serviceAccountName": "release-service-account"
+                  }
+                }
+              }
+              EOF
+    - name: run-task
+      taskRef:
+        name: rh-sign-image
+      params:
+        - name: requester
+          value: testuser-single
+        - name: pipelineRunUid
+          value: $(context.pipelineRun.uid)
+        - name: snapshotPath
+          value: snapshot_spec.json
+        - name: dataPath
+          value: data.json
+        - name: releasePlanAdmissionPath
+          value: release_plan_admission.json
+        - name: pyxisSecret
+          value: test-create-pyxis-image-cert
+        - name: taskGitUrl
+          value: "http://localhost"
+        - name: taskGitRevision
+          value: "main"
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: check-result
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+
+              internalPipelineruns="$(kubectl get pr \
+                -l "internal-services.appstudio.openshift.io/pipelinerun-uid=$(context.pipelineRun.uid)" \
+                -o json --sort-by=.metadata.creationTimestamp | jq -c)"
+              plrsLength=$(jq ".items | length" <<< "${internalPipelineruns}" )
+
+              if [ "${plrsLength}" != "0" ] ; then
+                echo "Error: seems pipelineruns were created, but signatures were supposed to be skipped"
+                exit 1
+              fi
+      runAfter:
+        - run-task
+  finally:
+    - name: cleanup
+      taskSpec:
+        steps:
+          - name: delete-crs
```

Comment:
I would think this task still has delete all internal plr (in case something went wrong with the test) or this task is removed entirely. Currently it just has a set which doesn't make sense

Reply:
corrected task name and implemented plr deletion.